src/status_im/ui/screens/discover/views.cljs

```diff
@@ -96,6 +97,41 @@
                                          :current-account current-account}]])]
      [react/text (i18n/label :t/none)])])
 
+(def public-chats-mock-data
+  [{:name  "Status team"
+    :count 25
+    :color "#B2F3E3"}
+   {:name  "ETH news"
+    :count 12
+    :color "#F7A7E8"}
+   {:name  "All about Ethereum"
+    :count 32
+    :color "#C1B8F0"}])
+
+(defn render-public-chats-item [item]
+  [react/view styles/public-chats-item-container
+   [react/view styles/public-chats-icon-container
+    [react/view (styles/public-chats-icon (:color item))
+     [react/text {:style styles/public-chats-icon-text}
+                 (-> item :name first str)]]]
+   [react/view styles/public-chats-item-inner
+    [react/view styles/public-chats-item-name-container
+     ;TODO(goranjovic) lightgray intentionally hardcoded while only a teaser
+     ;will be removed and properly styled when enabled
+     [vi/icon :icons/public {:color "lightgray"}]
```

Comment:
Also, it's probably not a big deal, but we are using keywords for colors and other props values in different places in code,  so i personally would use keywords instead of string for this purpose.

Reply:
I agree. In fact keywords are my personal preference as well. In this case, however it is not the same as `vi/icon` behaves differently for keywords. It has an explicit map of colors by keyword and if you provide a different keyword it defaults to `styles/icon-dark-color`. I wanted to force color `"lightgrey" ` and that's why I used a string.

Reply:
I'd rather not mess with `vi/icon` in this PR. What I could do is add a comment explaining this and maybe we can find a more general solution in the scope of another issue. It definitely would be good to get to a more consistent way. What do you think?

Reply:
so probably we just need to have `(or color styles/icon-dark-color)` here and that's it
https://github.com/status-im/status-react/blob/69a6baf09bcf5dafd724ab1b27ae01dfaf663488/src/status_im/components/icons/vector_icons.cljs#L102

Reply:
It can be even simpler. We only need the special behavior for some keywords that have an "overridden" color. Case can be replaced with a map of keywords and colors (in a `def` above) and then the whole thing is just `(that-map color color)`

Reply:
Because in the after state, it would only default to the dark color if `color` is nil or false. Previously, the same thing would've happened for unlisted keywords and any non-string args. We'd need to check all usages of that function when we make the change.